source/documentation/js-api.html.md.erb

```diff
@@ -832,6 +832,39 @@ h1 {
 });
 ```
 
+### Other
+
+#### `quietDeps`
+
+<% impl_status dart: '1.35.0', node: false %>
+
+If this option is set to `true`, Sass won't print warnings that are caused by
+dependencies. A "dependency" is defined as any file that's loaded through a
+[load path] or an [importer]. Stylesheets that are imported relative to the
+entrypoint are not considered dependencies.
+
+[load path]: #includepath
+
+This is useful for silencing deprecation warnings that you can't fix on your
+own. However, it's a good idea to *also* make sure that your dependencies know
```

Comment:
nit: this could be a stronger call-to-action—maybe "file issues with" or "notify your dependencies of"?

Reply:
Good idea, done.